source/_integrations/cloud.markdown

```diff
@@ -16,7 +16,7 @@ ha_platforms:
 ha_integration_type: system
 ---
 
-The Home Assistant Cloud allows you to quickly integrate your local Home Assistant with various cloud services like Amazon Alexa and Google Assistant. [Learn more.](/cloud)
```

Comment:
This change is incorrect.

> Home Assistant Cloud has nothing to do with Google or Alexa.

It does :)

../Frenck

Reply:
Ok, I did write the description of this PR too quickly. I still believe the [`cloud:` configuration page](https://www.home-assistant.io/integrations/cloud/) isn't clear and should be updated. Personally, the current description made me think that this configuration somehow allowed Home Assistant to *directly* interface with Alexa and Google Home.

It's not what this configuration does : this page is about enabling a service called "Home Assistant Cloud" that has several benefits, some of which are indeed related to Alexa and Google Home.

If you look at [NabuCasa's website](https://www.nabucasa.com/), you'll see that Home Assistant Cloud's **main** selling point is "Control your Home Assistant from anywhere.", **not** the Alexa or Google Home integration. I think this should be reflected in the [`cloud:` integration page](https://www.home-assistant.io/integrations/cloud/)

If you think the "Google Home and Alexa" bit should stay (which I absolutely can agree with), maybe we could simply make the description a little bit longer to reflect's NabuCasa's copy? How about this:

---

### Home Assistant Cloud
Home Assistant Cloud is a subscription service offered by NabuCasa that allows you to control your Home Assistant from outside your home network and to control Home Assistant entities from either Amazon Alexa or Google Home. [Learn more.](https://www.nabucasa.com/config/)